testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -412,6 +412,84 @@ QUnit.test('From renders the right types of editors according to stylingMode opt
     assert.ok($testContainer.find('.dx-field-item .dx-textbox').hasClass('dx-editor-underlined'), 'right class rendered');
 });
 
+QUnit.module('Checkbox editor value', () => {
+    function checkCheckbox(editor, newBoolValue) {
+        const expectedValue = newBoolValue === 'not set' ? false : newBoolValue;
+        QUnit.assert.equal(editor.option('value'), expectedValue, `editor has ${expectedValue} value`);
+
+        const $checkBox = $(editor.element()).closest('.dx-checkbox');
```

Comment:
"$checkBox.hasClass" calls seem to be internal CheckBox implementation details and it should be enough to check the "editor.option('value')" only.

Reply:
It's jquery wrapper method.
I have use it to check the visual representation of the checkbox and check that it is synced with editor value

Reply:
outdated